recipes/qwt/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+import os
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.33.0"
+
+class QwtConan(ConanFile):
+    name = "qwt"
+    license = "LGPL-2.1-or-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qwt.sourceforge.io/"
+    topics = ("conan", "archive", "compression")
+    description = (
+        "The Qwt library contains GUI Components and utility classes which are primarily useful for programs "
+        "with a technical background. Beside a framework for 2D plots it provides scales, sliders, dials, compasses, "
+        "thermometers, wheels and knobs to control or display values, arrays, or ranges of type double."
+    )
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
```

Comment:
Do we want to follow https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#recommended-feature-options-names?